metabasepy/client.py

```diff
@@ -190,61 +194,174 @@ def get(self, card_id=None):
             url=url,
             headers=self.prepare_headers(),
             verify=self.verify,
-            proxies=self.proxies
+            proxies=self.proxies,
         )
         Resource.validate_response(response=resp)
         return resp.json()
 
-    def get_by_collection(self, collection_slug):
```

Comment:
This is used by the helper scripts:
- exporter.py
- migrator.py
although they are a bit old, i suggest to keep this method.